backend/setup.go

```diff
@@ -7,12 +7,24 @@ import (
 	"os"
 )
 
+const (
+	// PluginProfilerEnv is a constant for the GF_PLUGINS_PROFILER environment variable used to enable pprof.
+	PluginProfilerEnv = "GF_PLUGINS_PROFILER"
+
+	// PluginProfilerPortEnv is a constant for the GF_PLUGINS_PROFILER_PORT environment variable use to specify a pprof port (default 6060).
+	PluginProfilerPortEnv = "GF_PLUGINS_PROFILER_PORT"
```

Comment:
nit. Having constants always creates one extra navigation for me. For configuration names like this, I don't really see the point.

Reply:
mostly godoc